allenact/base_abstractions/task.py

```diff
@@ -407,6 +408,11 @@ def __init__(
         self.tasks = [task_classes[0](env=env, sensors=sensors, task_info=task_info, max_steps=max_steps, batch_index=0, **kwargs)]
         self.tasks[0].batch_index = 0
 
+        self.frames = None
```

Comment:
I see why you're doing this, but I guess I had a thought on overloading the render() command, so that it produces all the observations we might want to extract from the environment (including, for example, also expert actions). I'm not sure we should be explicit at this level about which type of data are rendered, but I'm happy to discuss it.